source/config/__init__.py

```diff
@@ -178,6 +178,7 @@ def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	reportPage = boolean(default=true)
 	reportLineNumber = boolean(default=False)
 	reportLineIndentation = boolean(default=False)
+	reportToneIndentation = boolean(default=False)

```

Comment:
I think we should rename this to `reportLineIndentationWithTones` or `lineIndentationTones`, since it isn't clear that this is related to line indentation (as opposed to paragraph indentation).